FoundationBuilder/src/Application.cpp

```diff
@@ -3,7 +3,27 @@
 // Initialize with command line arguments
 void Application::init(int argc, char** argv)
 {
+    CommandLine cl;
+    cl.add(Option("InputModel", "specifies file path to input stl", "i", false, "input.stl"));
+    cl.add(Option("OutputModel", "specifies file path to output stl", "o", false, "output.stl"));
+    cl.add(Option("MetaData", "specifies file path to  metadate json", "m", false, "metadate.json"));
+    cl.add(Option("Height", "this is the foundation height", "h", false));
```

Comment:
Why the `height` do not have a default value? Should it be a `1.0`?

Reply:
@okrdima please check the specification and requirements, do they specify default value for height?

Reply:
here is a screenshot of documentation:
![image](https://user-images.githubusercontent.com/45259941/83631094-a2ef4c80-a5a5-11ea-9c81-493e7c1ee3eb.png)